content/en/Platform Deep Dive/Collaboration/Organization/Organization Settings/enforce-2fa.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Enforce Two-Factor Authentication"
+linkTitle: "Enforce 2FA"
+weight: 10
+description: >
+  Enforce 2FA for users.
+---
+
+{{% pageinfo %}}
+As an [Organization Owner](/getting-started/glossary/#organization-owner), you can enforce two-factor-authentication (2FA) for all users within your organization.
+{{% /pageinfo %}}
+
+2FA enforcement affects users belonging to your organization, including the following roles:
+
+{{% owner-member-team-member %}}
+
+{{< alert title="Note" color="primary" >}}
+If your organization uses [SAML-based single sign-on (SSO)](/getting-started/sign-in/#saml-sso), you bypass any 2FA enforcement. Your SAML provider may already require 2FA.
+{{< /alert >}}
+
+## Enforce 2FA
+
+1. Navigate to **Settings** > **Identity & Access**.
+1. Under **Two-Factor Authentication**, select **Enforce 2FA**.<br><br>
+  ![Enforce 2FA for your organization](/deepdive/Enforce2FA.png "Enforce 2FA for your organization")
+1. In the overlay that appears, select **Yes, enforce 2FA** to confirm.
```

Comment:
Would it be helpful to have a visual here? I know that causes overhead in the case we update anything (which will will).

Reply:
@molfinn > Would it be helpful to have a visual here? I know that causes overhead in the case we update anything (which will will).
IMO, we shouldn't add too many visuals, especially for each step in the instruction. Many writers would say that even one visual per instruction is "too much."

> Is overlay the right word here? or confirmation modal.

An _overlay_ is a generic word for an element that covers a part of the screen. I'm using _overlay_ for simplicity and consistency to avoid multiple other terms like _pop-up_, confirmation _modal_, _dialog_ (not recommended).

I checked several UX resources, some designers use these terms interchangeably. In some design systems, a [modal](https://polaris.shopify.com/components/modal) is described as "an overlay that ..."